po/ja.po

```diff
@@ -21516,6 +20297,122 @@ msgid ""
 "```"
 msgstr ""
 
+#~ msgid ""
+#~ "```shell\n"
+#~ "$ cargo new exercise\n"
+#~ "     Created binary (application) `exercise` package\n"
+#~ "```"
+#~ msgstr ""
+#~ "```shell\n"
+#~ "$ cargo new exercise\n"
+#~ "     Created binary (application) `exercise` package\n"
+#~ "```"
+
+#~ msgid ""
+#~ "```shell\n"
+#~ "$ cd exercise\n"
+#~ "$ cargo run\n"
+#~ "   Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+#~ "    Finished dev [unoptimized + debuginfo] target(s) in 0.75s\n"
+#~ "     Running `target/debug/exercise`\n"
+#~ "Hello, world!\n"
+#~ "```"
+#~ msgstr ""
+#~ "```shell\n"
+#~ "$ cd exercise\n"
+#~ "$ cargo run\n"
+#~ "   Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+#~ "    Finished dev [unoptimized + debuginfo] target(s) in 0.75s\n"
+#~ "     Running `target/debug/exercise`\n"
+#~ "Hello, world!\n"
+#~ "```"
+
+#~ msgid ""
+#~ "```shell\n"
+#~ "$ cargo run\n"
+#~ "   Compiling exercise v0.1.0 (/home/mgeisler/tmp/exercise)\n"
+#~ "    Finished dev [unoptimized + debuginfo] target(s) in 0.24s\n"
+#~ "     Running `target/debug/exercise`\n"
```

Comment:
Feel free to delete these old translations. They're put in a kind of "paper bin" by `msgmerge`, but they should be deleted when they don't look useful.